cocos/2d/renderer/static-vb-accessor.ts

```diff
@@ -51,14 +51,22 @@ const _entryPool = new Pool<IFreeEntry>(() => ({
  */
 export class StaticVBChunk {
     // public ib: Uint16Array;
-    constructor (
+    //
+    public meshBuffer: MeshBuffer = null!;
+    public indexBuffer: Uint16Array = null!;
+
+    constructor(
         public vertexAccessor: StaticVBAccessor,
         public bufferId: number,
         public vertexOffset: number,
         public vb: Float32Array,
         indexCount: number,
+        mb: MeshBuffer,
+        ib: Uint16Array
```

Comment:
Should remove `vb` and `ib` property, only pass `meshBuffer` as parameter and access vb & ib through `meshBuffer.vData` & `meshBuffer.iData`